04.ls/ls.rb

```diff
@@ -1,14 +1,40 @@
+#!/usr/bin/env ruby
+
 # frozen_string_literal: true
 
+require 'optparse'
+
 COL_COUNT = 3
 
 def main
-  entry_names = Dir.glob('*')
+  options = parse_options
```

Comment:
```suggestion
  success, options = parse_options
```

戻り値を2つの変数で受け取る方がスマートだと思います

Reply:
こちら修正しました！